joinpoints/src/main/java/com/fibermc/joinpoints/JoinpointsCommandRegistry.java

```diff
@@ -0,0 +1,95 @@
+package com.fibermc.joinpoints;
+
+import com.fibermc.joinpoints.commands.*;
+import com.mojang.brigadier.arguments.BoolArgumentType;
+import com.mojang.brigadier.arguments.StringArgumentType;
+import com.mojang.brigadier.tree.LiteralCommandNode;
+import net.fabricmc.fabric.api.command.v2.CommandRegistrationCallback;
+import net.minecraft.command.argument.EntityArgumentType;
+import net.minecraft.server.command.CommandManager;
+import net.minecraft.server.command.ServerCommandSource;
+
+import static net.minecraft.server.command.CommandManager.argument;
+
+public final class JoinpointsCommandRegistry {
+    private JoinpointsCommandRegistry() {}
+
+    public static void register() {
+        CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> {
```

Comment:
we should consider registering on the EC root node, and also supporting `excluded_top_level_commands`